test/libsolidity/syntaxTests/structs/member_type_eq_name_2.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
+    enum E {a, b, c}
+    struct S {E X; uint E;}
+    struct T {E T; uint E;}
+    struct U {E E;}
+}
```

Comment:
Should this test pass?

Reply:
According to my understanding, yes, because we allow struct members to have same names as types.